ros_docker_images/templates/docker_images/create_base_image.Dockerfile.em

```diff
@@ -26,10 +19,16 @@ ENV TZ @timezone
 
 # install bootstrap tools
 RUN apt-get update && apt-get install -q -y \
+    locales \
     python-rosdep \
     python-rosinstall \
     python-vcstools
 
+# setup environment
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV TZ @timezone
```

Comment:
hmmm that's a good question, TBH I just cut-n-paste without looking into it. seems that this has been around since the initial commit https://github.com/osrf/docker_templates/commit/b8e1a67246c7c4816b30efbee62ff201c78d79e9#diff-ac3c155fc308c7e6f7e2cd73fa2fafc4R12. Not sure it's needed though